WifiART/src/userinterface/BssidView.java

```diff
@@ -55,8 +58,100 @@ private void setup() {
         this.add(bssidListInstructionsLabelPanel);
 
         bssidListPanel.setLayout(new BoxLayout(bssidListPanel, BoxLayout.Y_AXIS));
+        bssidListPanel.setPreferredSize(new Dimension(1000, 300));
```

Comment:
Make the numbers PANEL_X_DIM and PANEL_Y_DIM (or similar) constants at the top of the program